### DISCUSSION TITLE
Welcome to POPForums Discussions!

### DISCUSSION THREAD

**Discussion by jeffputz** [POPWorldMedia/POPForums#214] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by afarazit** [comment]
Great idea, a discussions place is always useful.
